openml/extensions/sklearn/extension.py

```diff
@@ -476,6 +477,35 @@ def _is_sklearn_flow(cls, flow: OpenMLFlow) -> bool:
             or ',sklearn==' in flow.external_version
         )
 
+    def _get_sklearn_description(self, model: Any, char_lim: int = 1024) -> str:
+        '''Fetches the sklearn function docstring for the flow description
+
+        Parameters
+        ----------
+        model: The sklearn model object
+        char_lim: int, specifying the max length of the returned string
```

Comment:
Same here. Also, please put only the data type in the first line and put the description in the following lines.